lib/modules/orangered.js

```diff
@@ -205,48 +271,112 @@ const setupFaviconBadge = _.once(() => {
 const orangeredElements = () => ({
 	...nativeButtons(),
 	...floatingButtons(),
-	modmailButtonSelector: '#modmail',
 });
 
 const $orangeredElements = () => Object.values(orangeredElements()).reduce(($all, ele) => $all.add(ele), $());
 
-const nativeButtons = _.once(() => {
-	const nativeInboxButton = document.querySelector('#header-bottom-right #mail');
-	let nativeInboxCount = document.querySelector('#header-bottom-right .message-count');
-	if (!nativeInboxCount && (module.options.updateCurrentTab.value || module.options.updateOtherTabs.value)) {
+const nativeButtons = _.once(() => _nativeButtons(document.body, true));
+const _nativeButtons = (container = document.body, addIfMissing) => {
+	const nativeMessageButton = container.querySelector('#header-bottom-right #mail, #header-bottom-right #message');
```

Comment:
These shouldn't be inlined because some of the elements are used to find other buttons elements in `getOrAddInboxCount`.  It also makes it easier to copy-paste the object for destructuring later.